tasks/user.yml

```diff
@@ -50,7 +50,7 @@
     - candlepin-user
 
 - name: Build & Deploy the candlepin server
-  shell: "./server/bin/deploy {{candlepin_deploy_args}}"
+  shell: "./bin/deployment/deploy {{candlepin_deploy_args}}"
```

Comment:
@wolfdale This will make it so that `vagrant up` will only work with master branch. Can we make this conditional on whether the deployment dir exists, and if not, running with the old path?

Reply:
I don't think yaml file support conditional statements. I could find other alternatives.

My assumption was that we perform ```vagrant up``` on master branch only & then switch branches that we want to work with. Since no matter which branch we are, only latest ansible roles will get picked. This is the reason were we want to have multiple vagrant files & start tagging ansible-role repo with each release.

Reply:
you could make a conditional statement within the shell command perhaps, but you are right. we only support master for now.

Let's make that clear by adding a note in the README.md of this project. (that this ansible role only works with master candlepin, and you would have to manually switch branch and re-configure/run candlepin as needed for older branches)